DESCRIPTION

```diff
@@ -31,11 +31,13 @@ Depends:
     R (>= 3.5)
 Encoding: UTF-8
 RoxygenNote: 7.1.2
+Config/testthat/edition: 3
 Roxygen: list(markdown = TRUE)
 Suggests: 
     knitr,
     rmarkdown,
-    testthat (>= 2.1.0),
+    testthat (>= 3.0.0),

```

Comment:
@Yuri05 We can use those packages even if they are not part of the validR env right? It's just for testing so not a direct dependencies

Reply:
Yes, absolutely!

I would be happy to chat about this in person and maybe also do a live demo of these testing frameworks and how they would play out for our packages.